ui-v2/src/components/concurrency/global-concurrency-view/data-table/data-table.tsx

```diff
@@ -53,11 +59,36 @@ export const GlobalConcurrencyDataTable = ({
 	onEditRow,
 	onDeleteRow,
 }: Props) => {
+	const navigate = routeApi.useNavigate();
+	const { search } = routeApi.useSearch();
```

Comment:
I think including a dependency on the router might make it harder to test this component, but I'm not sure, and we can refactor if that turns out to be the case.